code/parachain/frame/composable-traits/src/dex.rs

```diff
@@ -18,6 +18,8 @@ use sp_runtime::{
 use sp_std::{collections::btree_map::BTreeMap, fmt::Debug, ops::Mul, vec::Vec};
 
 /// Specifies and amount together with the asset ID of the amount.
+// REVIEW(benluelo): Evaluate the usefulness of this type. We often end up converting between
```

Comment:
The reason this exists is to not mixup asset amounts by mistake(pretty common) in math operations and always return the assetId as part of an amount. Must not remove